event-streams/jmxtrans/README.md

```diff
@@ -0,0 +1,180 @@
+# JmxTrans
+
+[JMX Trans](https://github.com/jmxtrans/jmxtrans) is a tool for collecting data from JMX endpoints of Java applications and sending it to other applications and services. 
+
+The ability to deploy JmxTrans was deprecated in Event Streams 11.1.5 and removed in Event Streams 11.2.0. While the JmxTrans tool remains unsupported, this document provides high-level guidance on how to deploy it separately alongside an Event Streams instance.
+
+## JmxTrans Configuration
+
+[JmxTrans documentation](https://github.com/jmxtrans/jmxtrans/wiki) provides further guidance on defining [queries](https://github.com/jmxtrans/jmxtrans/wiki/Queries) and [output writers](https://github.com/jmxtrans/jmxtrans/wiki/OutputWriters). The configuration depends largely on what information is to be captured and where it must be sent. 
+
+The configuration can be deployed as a ConfigMap and then made available to the JmxTrans tool. An example ConfigMap definition is included as a reference. 
+
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: <jmxtrans-config>
+  namespace: <namespace>
+  labels:
+    app.kubernetes.io/name: jmx-trans
+data:
+  config.json: |
+    '{
+      "servers": [
+        {
+          "host": "<eventstreams-name>-kafka-<broker-id>.<eventstreams-name>-kafka-brokers",
+          "port": 9999,
+          "queries": [
+            {
+              "obj": "kafka.server:type=BrokerTopicMetrics,name=*",
+              "attr": [
+                "Count"
+              ],
+              "outputWriters": [
+                {
+                  "@class": "com.googlecode.jmxtrans.model.output.StdOutWriter"
+                }
+              ]
+            }
+          ]
+        },
+        // repeat for each broker
+      ]
+    }'
+```
+
+Where:
+- `<jmxtrans-config>` is the name of the ConfigMap
+- `<namespace>` is the namespace where Event Streams is installed
+- `<eventstreams-name>` is the name of the Event Streams instance
+- `<broker-id>` is Kafka broker ID (for example 0) 
+
+
+## JmxTrans Container Image
+
+JmxTrans is a Java application, so a container image that includes a Java runtime and the JmxTrans tool is needed to deploy it on Kubernetes environments. JmxTrans does not support Java 17 yet, so Java 11 is required. We recommend that you build a custom image that include a Java runtime and the JmxTrans tool.
```

Comment:
```suggestion
JmxTrans is a Java application. This means that a container image, which includes a Java runtime and the JmxTrans tool, is needed to deploy the application on Kubernetes environments. Build a custom image that include a Java runtime and the JmxTrans tool.

**Important:** JmxTrans only supports Java 11.
```

Reply:
> JmxTrans does not support Java 17 yet, so Java 11 is required.

This line in its current position doesn't add any value to the paragraph. Should this be modified?